apacheconfig/lexer.py

```diff
@@ -271,6 +274,11 @@ def t_multiline_OPTION_AND_VALUE(self, t):
         if not process:
             return
 
+        if t.type == "OPEN_TAG":
+            if value.endswith("/>"):
```

Comment:
I think `/    >` would not be a valid open/close tag, right? Just to make sure.

Reply:
Nope, it would not be valid! Just tried it out with both `Config::General` and the native Apache parser.